templates/engage/de/what-can-you-do-with-kubernetes.html

```diff
@@ -1,8 +1,4 @@
-  {% extends "engage/base_engage.html" %}
-
-  {% block meta_description %}Ein umfassenden Überblick, Demos und Ideen für Kubernetes den praktischen Einsatz.{% endblock %}
-
-{% block title %}Wofür können Sie Kubernetes einsetzen?{% endblock %}
+{% extends_with_args "engage/base_engage.html" with title="Wofür können Sie Kubernetes einsetzen?" meta_image="https://assets.ubuntu.com/v1/73fe5532-business+icon.svg" meta_description="Ein umfassenden Überblick, Demos und Ideen für Kubernetes den praktischen Einsatz." %}
```

Comment:
This image is white and so wont show up well socially. Maybe remove it for now or take a screenshot of the size and upload a png?